modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentItemLocalServiceImpl.java

```diff
@@ -51,6 +51,30 @@
 public class CommerceShipmentItemLocalServiceImpl
 	extends CommerceShipmentItemLocalServiceBaseImpl {
 
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public CommerceShipmentItem addCommerceDeliverySubscriptionShipmentItem(
```

Comment:
We should name this,
```suggestion
	public CommerceShipmentItem addDeliverySubscriptionCommerceShipmentItem(
```